src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsSolver.hpp

```diff
@@ -105,8 +105,19 @@ class MultiphasePoromechanicsSolver : public CoupledSolver< SolidMechanicsLagran
 
   virtual void updateState( DomainPartition & domain ) override;
 
+  /*
+   * @brief Utility function to update the stabilization parameters at each time step
+   * @param[in] domain the domain partition
+   */
   void updateStabilizationParameters( DomainPartition & domain ) const;
 
+  /*
+   * @brief Utility function to set the stress initialization flag
+   * @param[in] performStressInitialization true if the solver has to initialize stress, false otherwise
+   */
+  void performStressInitialization( integer const performStressInitialization )
```

Comment:
This is a bit ambiguous since this member function does not `perform` anything. `setStressInitialization`?

Reply:
Done, I renamed this function `setStressInitialization` as you suggested